index.html

```diff
@@ -98,26 +98,23 @@ <h1>Upcoming Events</h1>
                             <div class="event-header">
                                 <!-- Title -->
                                 <div class="event-title">
-                                    <span class="event-type">Reading Group</span>
+                                    <span class="event-type">Poster Session</span>
                                     <span class="event-name">
-                                        <a href="http://langcog.stanford.edu/papers_new/goodman-2016-tics.pdf">
-                                        Pragmatic Language Interpretation</a>
+                                        EECS 442 Computer Vision Poster Session
                                     </span>
                                 </div>
                 
                                 <!-- Info -->
                                 <div class="event-info">
-                                    Tuesday (2019-10-08) at 18:00 in BBB 3725
+                                    Tuesday (2019-12-10) at 18:00 in BBB 3725
                                 </div>
                             </div>
                 
                             <div class="event-body">
                                 <p>
-                                    Traditional NLP models view language as a set of fixed conventions.
-                                    Pragmatics is the study of how language is used for communicative purposes.
-                                    We explore the rational speech act (RSA) model, which is a probabilistic model
-                                    that formalizes many intuitive aspects of language and enables us to predict
-                                    human behavior.
+                                    This week we will visit the EECS 442 computer vision poster session!
```

Comment:
Convert this to past tense

Reply:
made it future tense to reflect that it is the "upcoming event" since we don't technically have an upcoming event right now